src/content/quickstart/openshiftsd/_index.md

```diff
@@ -0,0 +1,166 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "OpenShift with Service Discovery (AWS)"
+weight: 15
+---
+
+## AWS
+
+### openshift-install and pull-secret
+
+Download the **openshift-install** and _oc_ tools, and copy your _pull secret_ from:
+
+> https://cloud.redhat.com/openshift/install/aws/installer-provisioned
+
+Find more detailed instructions here:
+
+> https://docs.openshift.com/container-platform/4.3/installing/installing_aws/installing-aws-default.html
+
+
+### Make sure the aws cli is properly installed and configured
+
+Installation instructions
+
+> https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html
+
+```bash
+$ aws configure
+AWS Access Key ID [None]: ....
+AWS Secret Access Key [None]: ....
+Default region name [None]: ....
+Default output format [None]: text
+```
+
+See also for more details:
+
+> https://docs.openshift.com/container-platform/4.3/installing/installing_aws/installing-aws-account.html
+
+### Create and deploy cluster A
+
+In this step you will deploy cluster A, with the default IP CIDRs
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.128.0.0/14 |172.30.0.0/16 |
+
+
+```bash
+openshift-install create install-config --dir cluster-a
+```
+
+```bash
+openshift-install create cluster --dir cluster-a
+```
+
+The create cluster step will take some time, you can create Cluster B in parallel if you wish.
+
+### Create and deploy cluster B
+
+In this step you will deploy cluster B, modifying the default IP CIDRs
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.132.0.0/14 |172.31.0.0/16 |
+
+
+```bash
+openshift-install create install-config --dir cluster-b
+```
+
+
+Change the POD IP network, please note it’s a /14 range by default so you need to use 
++4 increments for “128”, for example: 10.132.0.0, 10.136.0.0, 10.140.0.0, ...
+ 
+```bash
+sed -i 's/10.128.0.0/10.132.0.0/g' cluster-b/install-config.yaml
+```
+
+Change the service IP network, this is a /16 range by default, so just use +1 increments
+for “30”: for example: 172.31.0.0, 172.32.0.0, 172.33.0.0, ...
+
+```bash
+sed -i 's/172.30.0.0/172.31.0.0/g' cluster-b/install-config.yaml
```

Comment:
This step is only required for testing purposes, it's not a mandatory step for setting up service-discovery. Can you mention that ?

Reply:
This step is to change the default CIDR's, this is required until we have global net support.

Reply:
This is a quickstart noam, just an explanation of how to deploy an environment *from scratch*, it's not a production deployment guide. We can work later on those.

I can add notes for that in subsequent PRs.

Reply:
It was working for me , may be kubectl verision? I have applied the change.